backstory/src/main/java/com/google/sps/story/PromptManagerWordTools.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// s
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.sps.story;
+
+import com.google.sps.APINotAvailableException;
+import com.google.sps.story.data.*;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Facilitates API calls through a single object.
+ */
+public class PromptManagerWordTools {
```

Comment:
`Tools` sounds like a class with static utility methods. Is there a name (likely a singular noun) that better indicates what the responsibility of the instance is?

Reply:
Changed to PromptManagerAPIsClient